lib/config_test.go

```diff
@@ -45,3 +45,18 @@ func TestFlagsToConfig(t *testing.T) {
 	assert.Equal("myregion", cfg.RegionName)
 
 }
+
+func TestSettingsFileToConfig(t *testing.T) {
+	assert := require.New(t)
+	flags := flag.NewFlagSet("testSettingsFile", flag.PanicOnError)
+
+	cfg, err := flagsToConfig(flags)
+	assert.NoError(err)
+	assert.NoError(flags.Parse([]string{"-source=testdata/.s3deploy.yml"}))
```

Comment:
This is not a good location because 'testdata' contains the data used by `deployer_test.go`. Is there a location suited for the test file?